### ISSUE TITLE
Adrian's email contact not working?

### ISSUE THREAD

**Issue by thomas-fossati** [thomas-fossati/draft-psa-token#51]
Hi [Adrian](@adrianlshaw), while sending the claims registration request to the IANA designated experts I got this automatic reply from the IETF mailer, which seems to suggest your email address is not set correctly?  Please see:

```
This is the mail system at host ietfa.amsl.com.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                   The mail system

<[adrian.shaw@hp.com]()> (expanded from
    <[expand-draft-tschofenig-rats-psa-token@virtual.ietf.org]()>): host
    us-smtp-inbound-1.mimecast.com[207.211.30.221] said: 550 SPF Sender Invalid
    - envelope rejected - https://community.mimecast.com/docs/DOC-1369#550
    [cO-ppDN6NFCAr71co_FM0g.us301] (in reply to RCPT TO command)
```
cheers!

**Comment by thomas-fossati** [comment]
Note that it may also be a problem with the IETF mailer: there seems to be an SPF check that failed but it's not completely clear to me who are Alice and Bob here :-)

**Comment by adrianlshaw** [comment]
Thanks for making me aware! This is odd. HP uses a system called Mimecast for dealing with spam. I will see if there's any way for me to add the IETF mailer to an allowlist. This is odd because I don't have any issues with IETF mailing list traffic

**Comment by adrianlshaw** [comment]
I added expand-draft-tschofenig-rats-psa-token@virtual.ietf.org to my permitted senders. I'll also follow up with my IT dept regarding ietfa.amsl.com.

**Comment by thomas-fossati** [comment]
Thanks Adrian!

**Comment by thomas-fossati** [comment]
Unfortunately I'm still having problems with Adrian's address, this time from a different Mimecast host:
```
<[adrian.shaw@hp.com]()> (expanded from
    <[expand-draft-tschofenig-rats-psa-token@virtual.ietf.org]()>): host
    us-smtp-inbound-1.mimecast.com[205.139.110.242] said: 550 SPF Sender
    Invalid - envelope rejected -
    https://community.mimecast.com/docs/DOC-1369#550
    [FXfPkbIUNcyIr4ZjPneyIg.us322] (in reply to RCPT TO command)
```
:-(

**Comment by adrianlshaw** [comment]
How annoying. I guess the issue might be on arm.com SPF and Mimecast being too fussy. I'll open a pull request with my ACM.org email.

**Comment by thomas-fossati** [comment]
Closed by #52
